api/deviceflow.go

```diff
@@ -0,0 +1,28 @@
+package api
+
+type ApproveDeviceFlowRequest struct {
+	UserCode string `json:"userCode" example:"BDSD-HQMK"`
+}
+
+type StartDeviceFlowRequest struct {
+	ClientID string `form:"client_id"`
```

Comment:
We shouldn't include this client ID where we don't support third party login using this method. Seems like we only ignore this.